source/_integrations/bosch_alarm.markdown

```diff
@@ -45,19 +45,109 @@ This entity reports state (_disarmed_, _armed_away_, etc.).
 The primary means of authentication for the _Mode 2_ API is the _Automation_ passcode. It needs to be at least 10 characters long, and it is different from the _User_ code -- a shorter numeric pin used to arm/disarm the panel.
 The integration will prompt for the required passcodes, which depend on the panel type.
 
-| Panel | Code |
-| --- | --- |
-| Solution | User [^2] |
+| Panel    | Code       |
+| -------- | ---------- |
+| Solution | User [^2]  |
 | B Series | Automation |
 | G Series | Automation |
-| AMAX | Both |
+| AMAX     | Both       |
 
 [^2]: The user needs to have the "master code functions" authority if you wish to interact with history events.
 
 {% important %}
 Since the _Mode 2_ automation user has "superuser" privileges, it bypasses the regularly configured alarm pin: you will _not_ be prompted for a _User_ code when arming/disarming through the integration.
 {% endimportant %}
 
+## Data updates
+
+The Bosch Alarm Panel {% term integration %} fetches data from the device every 30 seconds.
+Newer devices and firmware revisions have the possibility to push data instead of needing to rely on {% term polling %}.
+At startup, the integration checks whether your panel supports push data updates and falls back to {% term polling %} if not.
+
+## Troubleshooting
+
+### Issues with Bosch Solution 2000/3000/4000 panels
+
+We have found that some panels end up with a configuration on them that is incompatible with the integration. 
+When this happens, you will get a connection error when attempting to communicate with the panel, even if the credentials are correct.
+If you continue to have connection errors even after rebooting your panel, then you have an issue with your configuration.
+The easiest way to solve this is to follow the full reset and restore procedure outline below.
+
+#### Resetting the configuration and restoring the integration
+
+1. Update firmware (Recommended).
+   1. Download and install the latest firmware for the control panel and IP module from the Bosch Security website.
+2. Back up the existing configuration.
+   1. Connect to the panel via A-Link Plus.
+   2. Perform an Upload of the panel configuration.
+   3. Save the configuration to your computer.
+3. Default the control panel.
+   1. Press the default/reset button on the panel.
+   2. Use installer code 1234.
+   3. Set:
+      1. Location 0081 = 3 (Enables IP module mode)
+      2. Location 4456 = 4 (Enables RSC+ communication)
+   4. Set date and time using master code `25806#`.
+4. Initial Home Assistant test.
+   1. Wait 2 to 5 minutes after resetting the panel.
+   2. Connect Home Assistant to the panel using its IP address.
+   3. Home Assistant should connect using default config and show panel status.
+5. Restore your original configuration.
+   1. Reconnect to the panel using A-Link Plus.
+   2. Modify zones, outputs, and user codes to match original setup.
+   3. Save and Download the updated config to the panel.
```

Comment:
Are these starting with a capital letter because they are UI strings? In this case, they should be bold. We use bold to mark UI strings.